### ISSUE TITLE
Question on Test Set Parameter Tuning Strategy for MOTChallenge Submission

### ISSUE THREAD

**Issue by LIU-coder79** [kamkyu94/TrackTrack#4]
```
    if 'MOT17' in vid_name:
        # Path
        if mode == 'val':
            args.pickle_path = args.pickle_dir + 'mot17_val_0.80.pickle'
            args.pickle_path_95 = args.pickle_dir + 'mot17_val_0.95.pickle'
            args.data_path = args.data_dir + 'MOT17/train/'
        else:
            args.pickle_path = args.pickle_dir + 'mot17_test_0.80.pickle'
            args.pickle_path_95 = args.pickle_dir + 'mot17_test_0.95.pickle'
            args.data_path = args.data_dir + 'MOT17/test/'

        if '01' in vid_name or '03' in vid_name or '12' in vid_name:
            args.det_thr, args.init_thr = 0.65, 0.75
        elif '07' in vid_name:
            args.det_thr, args.init_thr = 0.60, 0.60
        elif '14' in vid_name:
            args.det_thr, args.init_thr = 0.45, 0.55
        else:
            args.det_thr, args.init_thr = 0.60, 0.70
        args.match_thr = 0.70

    elif 'MOT20' in vid_name:
        if mode == 'val':
            args.pickle_path = args.pickle_dir + 'mot20_val_0.80.pickle'
            args.pickle_path_95 = args.pickle_dir + 'mot20_val_0.95.pickle'
            args.data_path = args.data_dir + 'MOT20/train/'
        else:
            args.pickle_path = args.pickle_dir + 'mot20_test_0.80.pickle'
            args.pickle_path_95 = args.pickle_dir + 'mot20_test_0.95.pickle'
            args.data_path = args.data_dir + 'MOT20/test/'

        if '08' in vid_name:
            args.det_thr, args.init_thr = 0.30, 0.40
        if '04' in vid_name or '06' in vid_name or '07' in vid_name:
            args.det_thr, args.init_thr = 0.40, 0.50
        else:
            args.det_thr, args.init_thr = 0.40, 0.40
        args.match_thr = 0.55
```
I've implemented some modifications to your approach that demonstrate improved performance on the MOT17 validation set. However, when evaluating these changes through the official MOTChallenge server, I observed significant performance degradation on the test set.
 I'm encountering challenges in tuning parameters for the MOT17 test set due to the strict submission limits on the MOTChallenge evaluation server. Since ground truth (GT) for the test set is unavailable and we can only obtain metrics through limited server submissions, it is difficult to iteratively optimize thresholds without risking overfitting or exhausting submission quotas.
However, I noticed in your code that specialized threshold parameters were used for the test set (e.g.,  in `TrackTrack/Tracker/utils/etc.py`). Could you clarify:
1. How were these test-specific parameters determined?
2. Do you have recommendations for tuning under submission constraints?
